spyder/dependencies.py

```diff
@@ -133,6 +134,10 @@
      'package_name': "pexpect",
      'features': _("Stdio support for our language server client"),
      'required_version': PEXPECT_REQVER},
+    {'modname': "applaunchservices",

```

Comment:
This should only be added on Mac. HAving this missing dependency on other OSs would be confusing.